libsrc/osic1p/bootstrap.s

```diff
@@ -45,6 +45,8 @@ GETCHAR         :=      $FFBF           ; gets one character from ACIA
 FIRSTVISC       =       $85             ; Offset of first visible character in video RAM
 LINEDIST        =       $20             ; Offset in video RAM between two lines
 
+        lda     #0
+        sta     load
```

Comment:
Ok, I will modify my fix with your streamlined implementation, and create a direct pull request to the main cc65 repository.